wormhole/aggregator/src/lib.rs

```diff
@@ -1,5 +1,12 @@
 pub mod aggregator;
-pub mod circuit;
+pub mod circuits;
+mod util;
 
 /// The maximum numbers of proofs to aggregate into a composite proof.
-pub const DEFAULT_NUM_PROOFS_TO_AGGREGATE: usize = 10;
+pub const DEFAULT_NUM_PROOFS_TO_AGGREGATE: usize = TREE_BRANCHING_FACTOR.pow(TREE_DEPTH);
```

Comment:
I think this formula is not quite right, for example if TREE_BRANCHING_FACTOR == 3 and we have TREE_DEPTH = 2, the number should be 9 + 3, right? and for branching factor 2, we have 4 + 2, so I think it's some geometric series 

SUM(i = 1, i <= TREE_DEPTH, TREE_BRANCHING_FACTOR.pow(i)) 

so something like

(TREE_BRANCHING_FACTOR.pow(TREE_DEPTH + 1) - TREE_BRANCHING_FACTOR) / (TREE_BRANCHING_FACTOR - 1)